src/settings/DisplaySettingsForm.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'react-final-form';
+
+import {
+  Button,
+  Checkbox,
+  Layout,
+  Pane,
+} from '@folio/stripes/components';
+
+import stripesFinalForm from '@folio/stripes/final-form';
+
+const DisplaySettingsForm = ({
+  handleSubmit,
+  label,
+  pristine,
+  submitting,
+}) => (
+  <form id="courses-display-settings-form" onSubmit={handleSubmit}>
+    <Pane
+      defaultWidth="fill"
+      fluidContentWidth
+      id="pane-courses-settings-display"
+      lastMenu={(
+        <Button
+          buttonStyle="primary"
+          disabled={(pristine || submitting)}
```

Comment:
Why the parentheses?

Reply:
Good question. I copied this over from ERM and absolutely don't remember!

Reply:
Kill 'em! :-)